field/select/classes/field.php

```diff
@@ -502,9 +502,9 @@ public function userform_mform_element($mform, $searchform, $readonly) {
         // End of: element values.
 
         $attributes = array();
-        $attributes['id'] = $idprefix;
```

Comment:
I trust you here, not sure what's this change about, but I imagine it's for a good reason :-)